d3a_interface/constants_limits.py

```diff
@@ -58,6 +58,8 @@ class GeneralSettings:
         MIN_TICK_LENGTH_S = 1
 
         REDIS_PUBLISH_FULL_RESULTS = False
+        # Allow orders to contain additional requirements and attributes
+        ENABLE_DEGREES_OF_FREEDOM = True
```

Comment:
Could you please add some details about this?
For this change I followed what I found in `GlobalConfig`: we're usually defining new constants there, and we populate their value with what we defined in `ConstSettings`. In a different comment, you proposed this differentiation:
- `GlobalConfig`/`SimulationConfig`: parameters that affect each area independently.
- `ConstSettings`: parameters that affect the whole simulation.

Since this parameter affects the whole simulation and can't be changed separately for each area, you would only keep it inside `ConstSettings` and completely remove it from `GlobalConfig`. Is this correct?

Reply:
Depends on how you decide to implement it, the comments were written under different discussions that's why they are seemingly conflicting. If you want to make it configurable per-area (now or sometime in the future - which as far as I recall was what we discussed in a comment), then GlobalConfig / SimulationConfig is the way to go. If you want it global for the whole simulation, then should be added in ConstSettings. A mix of the 2 (e.g. adding it in GlobalConfig and having the default value in ConstSettings) should be avoided

Reply:
@spyrostz I then moved it into `GlobalConfig` (https://github.com/gridsingularity/d3a-interface/pull/281/commits/543d4b1597ce4116706ce57a6a1548bda89af899)